handlers.go

```diff
@@ -3635,11 +3639,35 @@ func (s *Server) handleListCollections(c echo.Context, u *util.User) error {
 func (s *Server) handleAddContentsToCollection(c echo.Context, u *util.User) error {
 	coluuid := c.Param("coluuid")
 
-	var contentIDs []uint
-	if err := c.Bind(&contentIDs); err != nil {
+	// we accept both {"contentids": [1, 2]} and [1, 2] as json payloads
```

Comment:
@gmelodie why do we accept 2 formats, is it due to backward compatibility?

Reply:
See [Slack discussion](https://filecoinproject.slack.com/archives/C016APFREQK/p1668026579464679?thread_ts=1668005715.880389&cid=C016APFREQK)

Reply:
@alvin-reyes weight in

Reply:
Yes. backward compatibility. We want to support the new json format without negatively impacting any existing users of this endpoint.